src/app/pages/pages-menu.ts

```diff
@@ -174,3 +174,179 @@ export const MENU_ITEMS: NbMenuItem[] = [
     ],
   },
 ];
+
+export const MENU_ITEMS_AR: NbMenuItem[] = [
```

Comment:
In my opinion all translations should be *only* within the translation files. It doesn't make sense to have translated strings scathered all over the project... There are ways to translate the menu ([this](https://gist.github.com/dizco/0182af54fa90e9b1e4d7751c60a2a787) is one example)

Reply:
Will try to apply it :)